src/main/resources/META-INF/beans.spring.xml

```diff
@@ -20,8 +20,8 @@
         <property name="groupId" value="${pom.groupId}"/>
         <property name="artifactId" value="${pom.artifactId}"/>
         <property name="version" value="${pom.version}"/>
-        <property name="soapSpecVersion" value="1.1"/>
-        <property name="restSpecVersion" value="1.3"/>
+        <property name="soapSpecVersion" value="1.3"/>
```

Comment:
Are we sure that the version for the specification documente of SOAP is not still 1.1?

Reply:
If my understanding is correct, they are both to 1.4 now (I am making the changes right now in the docs and the code to account for this). It seems that there is a confusion between the version of the SOAP with the wsdl version.